client/client.go

```diff
@@ -12,6 +12,21 @@ import (
 // RawJSON controls if the client does json handling or outputs it raw
 var RawJSON = false
 
+func GenericJSONErrorHandling(resp *resty.Response, err error) (*resty.Response, error) {
+	if err == nil {
+		if resp.StatusCode() != 200 && resp.StatusCode() != 400 && resp.StatusCode() != 503 {
```

Comment:
Should we maybe also add 403 here? From what I can tell `APIForbidden` does return a JSON body with more specific error as well :thinking: